Services/data/devicemodels/truck-02.json

```diff
@@ -0,0 +1,64 @@
+{
+    "SchemaVersion": "1.0.0",
+    "Id": "truck-02",
+    "Version": "0.0.1",
+    "Name": "Faulty Truck",
+    "Description": "Truck with GPS, speed and cargo temperature sensors. Temperature is higher than normal.",
+    "Protocol": "MQTT",
+    "Simulation": {
+        "InitialState": {
+            "online": true,
+            "latitude": 47.25433,
+            "longitude": -121.177075,
+            "speed": 30.0,
+            "speed_unit": "mph",
+            "cargotemperature": 49.0,
+            "cargotemperature_unit": "F"
+        },
+        "Interval": "00:00:10",
+        "Scripts": [
+            {
+                "Type": "javascript",
+                "Path": "truck-02-state.js"
+            }
+        ]
+    },
+    "Properties": {
+        "Type": "Truck",
+        "Location": "Field",
+        "Latitude": 47.25433,
+        "Longitude": -121.177075
+    },
+    "Telemetry": [
+        {
+            "Interval": "00:00:10",
+            "MessageTemplate": "{\"latitude\":${latitude},\"longitude\":${longitude},\"speed\": ${speed},\"speed_unit\":\"${speed_unit}\",\"cargotemperature\":${cargotemperature},\"cargotemperature_unit\":\"${cargotemperature_unit}\"}",
+            "MessageSchema": {
+                "Name": "truck-sensors;v1",
+                "Format": "JSON",
+                "Fields": {
+                    "latitude": "double",
+                    "longitude": "double",
+                    "speed": "double",
+                    "speed_unit": "text",
+                    "cargotemperature": "double",
+                    "cargotemperature_unit": "text"
+                }
+            }
+        }
+    ],
+    "CloudToDeviceMethods": {
+        "FirmwareUpdate": {
+            "Type": "javascript",
+            "Path": "FirmwareUpdate-method.js"
+        },
+        "DecreaseCargoTemperature": {
+            "Type": "javascript",
+            "Path": "TBD.js"
```

Comment:
Is this supposed to be "TBD"?

Reply:
+1

Reply:
It was TBD from previous  version. I have added the actual file name, but again those files are all TODOs